### PR TITLE
Cleanup *.rc files

### PR DESCRIPTION
**Pull request by yacuken**

### CODE REVIEW

File: rootdir/etc/init.qcom-common.rc [hunk 1]
```diff
@@ -342,6 +342,11 @@ service sensors /system/bin/sensors.qcom
     user root
     group root
 
+service sensorservice /system/bin/sensorservice
+    class late_start
+    user root
+    group root
```
Comment (sledges) [inline rootdir/etc/init.qcom-common.rc:346]: would this not deserve to be in a separate commit? :)

Comment (yacuken) [reply rootdir/etc/init.qcom-common.rc:346]: ah. ok (:

**Review by sledges — APPROVED** [review]
sorry for the delay 😁
